docs/vertx-runtime/master.adoc

```diff
@@ -290,6 +258,55 @@ include::topics/secured-mission-resources.adoc[leveloffset=+3]
 //Add-ons
 //Building your app
 
+== Debugging
+
+This sections contains information about debugging your {VertX}&#x2013;based application both in local and remote deployments.
+
+=== Remote Debugging
+:context:
+
+To remotely debug an application, you must first configure it to start in a debugging mode, and then attach a debugger to it.
+
+
+:parameter-debug-property: debug.port
+:parameter-maven-goal: vertx:debug
+include::topics/proc_starting-your-application-locally-in-debugging-mode.adoc[leveloffset=+3]
+:parameter-debug-property!:
+:parameter-maven-goal!:
+
+//:parameter-uberjar-documented:
+//include::topics/proc_starting-an-uberjar-in-debugging-mode.adoc[leveloffset=+3]
+//:parameter-uberjar-documented!:
+
+include::topics/proc_starting-your-application-on-openshift-in-debugging-mode.adoc[leveloffset=+3]
+
+include::topics/proc_attaching-a-remote-debugger-to-the-application.adoc[leveloffset=+3]
+
+=== Debug Logging
+
+{VertX} provides a built-in logging API.
+The default logging implementation for {VertX} uses the `java.util.logging` library that is provided with the {ProductShortName} release of {VertX}.
```

Comment:
I'm not sure if I understand this sentence correctly, but `java.util.logging` is not provided with Vert.x, it's provided with Java :-)

Reply:
True. I have though one and wrote the other. let me correct that Thank you.